SPECS/vim/vim.spec

```diff
@@ -181,9 +178,6 @@ fi
 %{_datarootdir}/vim/vim*/colors/lists/default.vim
 %{_datarootdir}/vim/vim*/defaults.vim
 %{_datarootdir}/vim/vim*/filetype.vim
-%{_datarootdir}/vim/vim82/syntax/nosyntax.vim
-%{_datarootdir}/vim/vim82/syntax/syntax.vim
```

Comment:
These two files look like they're going to be moved to the `extras` subpackage. Can you confirm that the base package is still functional without these files after this change?

Reply:
Fixed this by adding the files back after updating their paths.